components/typography/Index.jsx

```diff
@@ -0,0 +1 @@
+<link rel="stylesheet" href="styles/global.css" />
```

Comment:
```suggestion
<link rel="stylesheet" href="styles/global.css" />

```
Файлы должны заканчиваться пустой строкой

Reply:
В целом содержание этого файла должно быть другим
```suggestion
import Paragraph from './Paragraph';
import Text from './Text';
import Title from './Title';

export type TypographyProps = {
  Text: typeof Text;
  Title: typeof Title;
  Paragraph: typeof Paragraph;
};

const Typography.Text = Text;
const Typography.Title = Title;
const Typography.Paragraph = Paragraph;

export default Typography;

```
Пример был в референсе

В нашем случае Index файлы в проекте позволяют сделать более красивый путь импорта компонента в дальнейшем